tests/kernel/common/src/printk.c

```diff
@@ -26,10 +25,21 @@ char *expected = "22 113 10000 32768 40000 22\n"
 		 "42 42   42       42\n"
 		 "42 42 0042 00000042\n"
 		 "255     42    abcdef        42\n"
-		 "ERR -1 ERR ffffffffffffffff\n"
+		 "68719476735 -1 18446744073709551615 ffffffffffffffff\n"
 ;
-#else
-
+#elif defined(CONFIG_CBPRINTF_COMPLETE)
+char *expected = "22 113 10000 32768 40000 %llu\n"
```

Comment:
Think this needs some adjustment, based on CI results.

Reply:
Yes, I'd missed the change to the pointer format in that test.